Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -63,13 +63,78 @@ public MainWindow(Settings settings, MainViewModel mainVM)
             InitSoundEffects();
 
             DataObject.AddPastingHandler(QueryTextBox, OnPaste);
+
+            this.Loaded += (obj, args) =>
+            {
+                var handle = new WindowInteropHelper(this).Handle;
+                var win = HwndSource.FromHwnd(handle);
+                win.AddHook(new HwndSourceHook(WndProc));
+            };
         }
 
+        DispatcherTimer timer = new DispatcherTimer
+        {
+            Interval = new TimeSpan(0, 0, 0, 0, 500),
+            IsEnabled = false
+        };
+
         public MainWindow()
         {
             InitializeComponent();
         }
 
+        private void OnSizeChanged(object sender, SizeChangedEventArgs e)
+        {
+
+        }
+
+        private const int WM_ENTERSIZEMOVE = 0x0231;
+        private const int WM_EXITSIZEMOVE = 0x0232;
+        public event EventHandler ResizeBegin;
+        public event EventHandler ResizeEnd;
+        private IntPtr WndProc(IntPtr hwnd, int msg, IntPtr wParam, IntPtr lParam, ref bool handled)
+        {
+            if (msg == WM_ENTERSIZEMOVE)
+            {
+                OnResizeBegin();
```

Comment:
I think this is not needed and could be removed?

Reply:
OK